servicex/transformer_manager.py

```diff
@@ -78,7 +78,7 @@ def create_job_object(request_id, image, chunk_size, rabbitmq_uri, workers,
                 client.V1VolumeMount(mount_path="/data", name='rootfiles'))
 
         # Compute Environment Vars
-        env = [client.V1EnvVar(name="BASH_ENV", value="/home/atlas/.bashrc")]
+        env = [client.V1EnvVar(name="BASH_ENV", value="/workdir/.bashrc")]
```

Comment:
Since we are changing this, I think `/servicex` might be more descriptive

Reply:
Fair enough. Changed the working directory here and in the xAOD transformer